task-sdk/src/airflow/sdk/bases/sensor.py

```diff
@@ -145,40 +145,38 @@ def _coerce_poke_interval(poke_interval: float | timedelta) -> timedelta:
             return poke_interval
         if isinstance(poke_interval, (int, float)) and poke_interval >= 0:
             return timedelta(seconds=poke_interval)
-        raise AirflowException(
-            "Operator arg `poke_interval` must be timedelta object or a non-negative number"
-        )
+        raise RuntimeError("Operator arg `poke_interval` must be timedelta object or a non-negative number")
```

Comment:
If we are changing it:

```suggestion
        raise ValueError("Operator arg `poke_interval` must be timedelta object or a non-negative number")
```

Or maybe a `TypeError`, but it's not 100% worth us separating TypeError (wrong type) from ValueError (negative number)